api/PROFILE.md

```diff
@@ -144,6 +144,65 @@ To check if the last requested command has completed, compare its timestamp with
 `last_cmd_timestamp` field from `LocalAppInfo` message, submitted by EVE in the request
 body of the API.
 
+### DevInfo
+
+Publish the current state of the device to the local server and optionally obtain
+a command to execute.
+
+POST /api/v1/devinfo
+
+Return codes:
+
+* Success; with a command to execute as defined in the response body: `200`
+* Success; without a command to execute: `204`
+* Not implemented: `404`
+
+Request:
+
+The request mime type MUST be "application/x-proto-binary".
+The request MUST have the body of a single protobuf message of type [LocalDevInfo](./proto/profile/local_profile.proto).
+Device publishes information repeatedly to keep the local server updated and to allow the server
+to submit commands for execution.
+Local server MAY throttle or cancel this communication stream by returning the `404` code.
+
+Response:
+
+The response MAY contain the body of a single protobuf message of type [LocalDevCmd](./proto/profile/local_profile.proto),
+encoded as "application/x-proto-binary".
+
+The requester MUST verify that the response payload (if provided) has the correct `server_token`.
+If the verification succeeds, then the timestamp is checked to determine whether
+or not the command has already been executed, and it not it is applied.
+
+Currently, the method allows to request the a graceful Shutdown (of all app instances)
```

Comment:
`the a` -> `a`

Reply:
Thanks let me fix that in a follow-on PR.